### ISSUE TITLE
error: binding types: type aws:alb/LoadBalancerType:LoadBalancerType must be an object, not a string

### ISSUE THREAD

**Issue by clstokes** [pulumi/tf2pulumi#222]
I get the error below for the code below. I'm not using `aws:alb/LoadBalancerType:LoadBalancerType` in the code and if I repeat the `tf2pulumi` command I get a different resource type as the error so it seems "random". If I comment out the `output` the error does not occur.

```
% tf2pulumi version
v0.9.0
```

#### Code

```hcl
provider "aws" {
    region  = "us-west-2"
}

resource "aws_vpc" "main" {
    cidr_block = "10.0.0.0/16"
}

output "vpc_id" {
    value = aws_vpc.main.id
}
```

#### Error

```
% tf2pulumi                  
error: binding types: type aws:alb/LoadBalancerType:LoadBalancerType must be an object, not a string
% tf2pulumi 
error: binding types: type aws:rds/EngineMode:EngineMode must be an object, not a string
% tf2pulumi 
error: binding types: type aws:ec2/InstancePlatform:InstancePlatform must be an object, not a string
% tf2pulumi 
error: binding types: type aws:applicationloadbalancing/IpAddressType:IpAddressType must be an object, not a string
% tf2pulumi 
error: binding types: type aws:ec2/ProtocolType:ProtocolType must be an object, not a string
% tf2pulumi 
error: binding types: type aws:applicationloadbalancing/LoadBalancerType:LoadBalancerType must be an object, not a string
% tf2pulumi 
error: binding types: type aws:rds/EngineType:EngineType must be an object, not a string
```

#### Steps to reproduce

Run `tf2pulumi` on the code above.

**Comment by MitchellGerdisch** [comment]
In case it helps, in my experience this is only happening when I do `tf2pulumi --target-language python`
If I use target-language of typescript it works.
Also, in case it helps, I'm able to reproduce this using the main.tf found in this repo's tests/terraform/aws/ec2 folder.

**Comment by wuqunfei** [comment]
```
 tf2pulumi --target-language typescript                                                                                       
error: binding types: type aws:lambda/Runtime:Runtime must be an object, not a string
```

**Comment by wuqunfei** [comment]
It looks very a general issue, anyone can continue or give some idea to fix it.

**Comment by komalali** [comment]
I'm working on a fix @wuqunfei, we have some outdated dependencies in tf2pulumi, should be resolved with https://github.com/pulumi/tf2pulumi/pull/207

**Comment by komalali** [comment]
This is fixed in v0.10.0 of tf2pulumi
